core/src/main/scala/com/dimafeng/testcontainers/FixedHostPortGenericContainer.scala

```diff
@@ -10,7 +10,8 @@ class FixedHostPortGenericContainer(imageName: String,
                                     classpathResourceMapping: Seq[(String, String, BindMode)] = Seq(),
                                     waitStrategy: Option[WaitStrategy] = None,
                                     exposedHostPort: Int,
-                                    exposedContainerPort: Int
+                                    exposedContainerPort: Int,
+                                    fileSystemBind: Seq[(String, String, BindMode)] = Seq()
```

Comment:
Sorry for the delay. Just realized that I haven't submitted another comment - would it make sense to introduce a case class instead of tuple? otherwise, we need to make it more easier to understand what first two string params are.

Reply:
It makes sense to have that, what do you think is the best place to put such `case class`? as a name I'm thinking about `FileSystemBinding`. Let me know your thoughts ;)

Reply:
Yes, I think it's reasonable